guides/05_chatbots/02_chat_interface_examples.md

```diff
@@ -1,144 +1,69 @@
-In this Guide, we go through several examples of how to use `gr.ChatInterface` with popular LLM libraries.
+# Using Popular LLM libraries and APIs
 
-## A `langchain` example
+In this Guide, we go through several examples of how to use `gr.ChatInterface` with popular LLM libraries and API providers.
 
-Let's start by using `langchain` on top of `openai` to build a general-purpose streaming chatbot application in 19 lines of code. You'll need to have an OpenAI key for this example (keep reading for the free, open-source equivalent!)
+We will cover the following libraries and API providers:
 
-```python
-from langchain.chat_models import ChatOpenAI
-from langchain.schema import AIMessage, HumanMessage
-import openai
-import gradio as gr
+* [Llama Index](#llama-index)
+* [LangChain](#lang-chain)
+* [OpenAI](#open-ai)
+* [Hugging Face `transformers`](#hugging-face-transformers)
+* [SambaNova](#samba-nova)
+* [Hyperbolic](#hyperbolic)
+* [Anthropic's Claude](#anthropics-claude)
+* [Groq](#groq)
+* [Llama.cpp](#llamacpp)
 
```

Comment:
just remember to remove these if you merge without adding them. also `#llamacpp` should be `#llama-cpp`